libs/site-semantic-theme/src/semantic/thecoin/globals/site.variables

```diff
@@ -325,7 +339,7 @@
 /*-------------------
      Page variables
 --------------------*/
-@pageTransitionDuration: 300;
+@pageTransitionDuration: 0.3s;
```

Comment:
We seem to have 2 different transition times in here.  Should be looked into eventually